tabs/mission_control.js

```diff
@@ -2100,7 +2109,7 @@ TABS.mission_control.initialize = function (callback) {
             if (selectedMarker) {
                 selectedMarker.setP2(Number($('#pointP2').val()));
                 mission.updateWaypoint(selectedMarker);
-                mission.update(singleMissionActive());
+                mission.update(singleMissionActive());setAlt
```

Comment:
setAlt ?

Reply:
Damn cat. I was performing a find, and she trod on the keyboard. I'll get that sorted.